chicago-short-notes-bibliography.csl

```diff
@@ -0,0 +1,4040 @@
+      <text variable="supplement-number"/>
+    </group>
+  </macro>
+  <macro name="label-version">
+    <group delimiter=" ">
+      <choose>
+        <if type="software">
+          <!-- short version label software (CMOS 14.169) -->
+          <label form="short" variable="version"/>
+        </if>
+        <else>
+          <label variable="version"/>
+        </else>
+      </choose>
+      <text variable="version"/>
+    </group>
+  </macro>
+  <macro name="label-version-capitalized">
+    <group delimiter=" ">
+      <choose>
+        <if type="software">
+          <!-- short version label software (CMOS 14.169) -->
+          <label form="short" text-case="capitalize-first" variable="version"/>
+        </if>
+        <else>
+          <label text-case="capitalize-first" variable="version"/>
+        </else>
+      </choose>
+      <text variable="version"/>
+    </group>
+  </macro>
+  <macro name="label-volume">
+    <group delimiter=" ">
+      <choose>
+        <if is-numeric="volume">
+          <label form="short" variable="volume"/>
+        </if>
+      </choose>
+      <text variable="volume"/>
+    </group>
+  </macro>
+  <macro name="label-volume-capitalized">
+    <group delimiter=" ">
+      <choose>
+        <if is-numeric="volume">
+          <label form="short" text-case="capitalize-first" variable="volume"/>
+        </if>
+      </choose>
+      <text text-case="capitalize-first" variable="volume"/>
+    </group>
+  </macro>
+  <!-- 1. Author (CMOS 13.74–86) -->
+  <macro name="author-bib">
+    <names variable="composer">
+      <name delimiter-precedes-last="always" name-as-sort-order="first"/>
+      <label form="short" prefix=", "/>
+      <substitute>
+        <names variable="author"/>
+        <!-- cf. `interview` model (CMOS 14.110); use `contributor` if `host` should be prioritized -->
+        <names variable="guest"/>
+        <names variable="host"/>
+        <choose>
+          <if type="song">
+            <names variable="performer"/>
+          </if>
+        </choose>
+        <choose>
+          <if type="classic">
+            <!-- contributors fall after the title of `classic` (CMOS 14.147) -->
+            <text macro="author-title-substitute-bib"/>
+          </if>
+          <else-if type="entry-dictionary" variable="container-title">
+            <!-- contributors fall after the title of unsigned reference entries (CMOS 14.130) -->
+            <text macro="author-title-substitute-container"/>
+          </else-if>
+          <else-if type="entry-encyclopedia" variable="container-title">
+            <text macro="author-title-substitute-container"/>
+          </else-if>
+        </choose>
+        <names variable="illustrator"/>
+        <choose>
+          <if match="none" type="standard">
+            <names variable="editor-translator"/>
+            <names variable="editor"/>
+            <names variable="translator"/>
+            <names variable="collection-editor"/>
+          </if>
```

Comment:
@icornelius Thank you for testing this! I am having trouble reproducing the problem that you are reporting. For example, from your description I would expect both the editors and translators to appear before the title in [Ferreras et al., _Democratize Work_](https://www.zotero.org/groups/2205533/test_items_library/collections/5V67EPX3/search/ferreras/titleCreatorYear/items/CG2B2D3B/item-list) (_CMOS_ 14.60), but it appears for me as I would expect. Could you please let me know whether the problem is appearing for you with this, and if so whether you are using the most recent version of Zotero? Feel free to send me an email if you can export some specific examples with which this occurs.

Reply:
See <https://github.com/jgm/citeproc/issues/159>.